flow/src/main/java/com/sequenceiq/flow/reactor/api/handler/HandlerEvent.java

```diff
@@ -0,0 +1,34 @@
+package com.sequenceiq.flow.reactor.api.handler;
+
+import reactor.bus.Event;
+
+public class HandlerEvent<T> {
+
+    private Event<T> event;
+
+    private int eventCounter;
```

Comment:
Should this be atomic?

Reply:
haven't modified this, it's extracted as it was as an inner class